cryptopan/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod cryptopan;
```

Comment:
Simply define all the code in `lib.rs` without splitting it into a separate module. Also note that module level docs (i.e. the `//!` will be used as documentation).